### PR TITLE
Change views for pylint conformity

### PR DESCRIPTION
**Pull request by svenseeberg**

### CODE REVIEW

File: src/news/views.py [hunk 1]
```diff
@@ -23,11 +23,17 @@ def index(request, language_code):  # pylint: disable=W0613
         end = (count * page)
         posts = posts[start:end]
     for item in posts:
-        result.append({'id': item.id, 'title': item.title, 'tags': [item.name for item in item.newscategory.all()], 'date': str(item.pub_date)})
+        result.append({'id': item.id, 'title': item.title,
+                       'tags': [item.name for item in item.newscategory.all()],
+                       'date': str(item.pub_date)})
     result_json = json.dumps(result)
     return HttpResponse(result_json, content_type="application/json")
```
Comment (thimmy687) [inline src/news/views.py:30]: why not use JsonResponse directly?

File: src/news/views.py [hunk 2]
```diff
@@ -39,6 +45,7 @@ def singlenews(request, news_id):
     }
     return HttpResponse(json.dumps(result), content_type="application/json")
```
Comment (thimmy687) [inline src/news/views.py:46]: see msg before

**Review by thimmy687 — CHANGES_REQUESTED** [review]

File: src/news/views.py (hunk 1 above)
Comment (svenseeberg) [reply src/news/views.py:30]: No reason at all. Just happened this way and changing it totally valid :)

File: src/news/views.py (hunk 2 above)
Comment (svenseeberg) [reply src/news/views.py:46]: save as above

**Pull request closed without merging**